internal/common/tools/tools.go

```diff
@@ -73,19 +74,15 @@ func IncrementIP(origIP, cidr string) (string, error) {
 	return ip.String(), nil
 }
 
-func gnbIdGenerator(i int) string {
+func gnbIdGenerator(i int, gnbId string) string {
 
-	var base string
-	switch true {
-	case i < 10:
-		base = "00000"
-	case i < 100:
-		base = "0000"
-	case i >= 100:
-		base = "000"
+	gnbId_int, err := strconv.Atoi(gnbId)
```

Comment:
gNB ID is hexadecimal, not decimal.
https://www.techplayon.com/5g-nr-cell-global-identity-planning/